web/less/completeMeeting.less

```diff
@@ -0,0 +1,111 @@
+@import 'base.less';
+html {
+  background-color: #131415;
+  mix-blend-mode: normal;
+  background-image: url("/images/vonage-v-logo-background.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 100%;
+  height: 100%;
+}
+
+.completeMeeting {
+  margin-left: 100px;
+  font-family: "Spezia", arial !important;
+  color: white;
+  .completeMeeting-box-container {
+    display: -ms-inline-flexbox;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-align: start;
+    align-items: flex-start;
+    -ms-flex-direction: row;
+    flex-direction: row;
+  }
+  .completeMeeting-box {
+    min-height: 222px;
+    float: left;
+    max-width: 1200px;
+    padding: 0 60px 10px 0;
+    margin: 27px 200px 20px 0px;
```

Comment:
The margin-right setting here (200px) makes the columns to narrow when the browser becomes narrow (say 700px wide). Maybe set this to `margin: 27px 0 20px 0` and change the max-width to `452px`.